src/Tribe/JSON_LD/Event.php

```diff
@@ -78,6 +78,16 @@ public function get_data( $posts = null, $args = array() ) {
 		$return = array();
 
 		foreach ( $posts as $i => $post ) {
+			// We may have been passed a post ID - let's ensure we have the post object
+			if ( is_numeric( $post ) ) {
+				$post = get_post( $post );
+			}
+
+			// If we don't have a valid post object, skip to the next item
+			if ( ! is_object( $post ) ) {
```

Comment:
If you want to change to use `instanceof WP_Post` otherwise it’s good!

Reply:
Good call, addressed